textpipe/operation.py

```diff
@@ -0,0 +1,97 @@
+# pylint: disable=too-few-public-methods
+"""
+Operation classes must be defined in this module.
+
+In a future version we might consider adding support for fully qualified paths
+when creating a Pipeline, e.g.:
+Pipeline['CleanText', 'my.org.package.OperationClass'])
+
+so that users do not have to put their Operation classes inside of this module.
+"""
+
+class Operation:
+    """
+    Base class for pipeline operations.
+    """
+    pass
+
+
+class CleanText(Operation):
```

Comment:
Maybe a file per class?

Reply:
I would like that. Maybe we can group them in some logical sense? For example, have a `textpipe/operation/clean.py` and `textpipe/operation/spacy.py`?

I guess we can add a `textpipe/operation/__init__.py` that `imports * from` these or use some smarter introspection?

Reply:
I agree (also see module comment). But propose this not be part of your CRQ but implemented in a future story.

Reply:
I'm ok with that!

Reply:
https://github.com/textpipe/textpipe/issues/38